packages/apollo-server-errors/src/index.ts

```diff
@@ -264,3 +264,15 @@ export function formatApolloErrors(
     }
   }) as Array<ApolloError>;
 }
+
+export function hasPersistedQueryError(errors: Array<Error>): boolean {
+  try {
```

Comment:
@abernix Yea this might be overkill especially given the use of TypeScript. It was more or less guarding against any bad data that may be passed in. Let me know if you'd like me to remove.

Reply:
Can we remove the `try` / `catch` but guard with `Array.isArray(errors)` prior to using `Array.prototype.some`?

Reply:
Just pushed a fix. Now using `Array.isArray` as suggested.